deploy/docker/Dockerfile

```diff
@@ -1,28 +1,44 @@
 FROM alpine:3.5
 
-LABEL maintainer="adundovi@gmail.com"
-LABEL description="Prepares a docker image \
-of pump.io server, http://pump.io/."
+LABEL maintainer Andrej Dundovic <adundovi@gmail.com>
+LABEL description "Stream server that does \
+most of what people really want from a social network."
 
 ENV LC_ALL en_US.UTF-8
 
+ENV PUMPIO_VERSION 3.0.2
+
 RUN apk add --no-cache \
-  nodejs \
-  git \
-  python \
-  make \
-  g++ \
-  graphicsmagick
-RUN npm install -g databank-mongodb
-RUN npm install -g pump.io
-RUN adduser -D -g "" pumpio
+    curl \
+    nodejs \
+    python \
+    make \
+    g++ \
+    git \
+    graphicsmagick \
+ && adduser -D -g "" pumpio \
+ && mkdir -p /etc/ssl/private \
+ && npm install -g databank-mongodb@0.19.2 \
+ && mkdir -p /usr/src \
+ && curl -fSL https://github.com/pump-io/pump.io/archive/v$PUMPIO_VERSION.zip -o pumpio.zip \
```

Comment:
It is not cargo-cult programming - it is a [documented](https://docs.docker.com/engine/reference/builder/#add) and intended behaviour if you don't trust my arguments:

> The <src> path must be inside the context of the build; you cannot ADD ../something /something, because the first step of a docker build is to send the context directory (and subdirectories) to the docker daemon.

Of course, there is a workaround, but I don't want to rape the procedure. Up to now, I have always used Docker for deployment, and I understand that context well, but, like I said, I will try to enter the mind of a "docker developer" to understand how to set it up properly and efficiently, probably by moving the dockerfile to the root of this repository as the first step.